src/components/common/Select/Select.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react';
+
+import clsx from 'clsx';
+import styles from './Select.module.css';
+
+interface Option {
+  value: string;
+  text: string;
+}
+
+interface SelectProps {
+  options: readonly Option[];
+  defaultText: string;
+
+  name: string;
+  id: string;
+  value: string;
+  isError: boolean;
+  isRequired?: boolean;
```

Comment:
사소한데, 기본값이 지정했다면 (특히 예상할 수 있는 기본값과 다르다면) 주석으로 정보를 제공해도 좋겠습니다.

```suggestion
  /** @defaultValue true */
  isRequired?: boolean;
```

Reply:
JSDoc에 기본값을 알려주는 주석이 있군요! 꿀팁 감사합니다 하루.
바로 적용했습니다! [[커밋]](https://github.com/woowacourse/react-payments/pull/390/commits/3d2ff549639eff819d9681a733f244f74565e33f)